starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -50,3 +50,11 @@ public void findAll() {
     Assert.assertFalse(actualInventory.isEmpty());
   }
 }
+//task1
+public void createDAOtest() {
+    Inventory inventory = new Inventory();
```

Comment:
We would also like to set the ID to test if `mongoTemplate` is setting a new ID.
Add a similar setter and constant (like NAME or PRODUCT_TYPE) to the inventory.